package.json

```diff
@@ -39,6 +39,7 @@
   "homepage": "https://github.com/GeoTIFF/georaster-layer-for-leaflet#readme",
   "dependencies": {
     "chroma-js": "^1.4.0",
+    "lodash": "^4.17.20",
```

Comment:
Could we remove the lodash dependency for the reason expressed above?  Thank you.